casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperReportingSpec.scala

```diff
@@ -17,24 +19,26 @@ class MultiParentCasperReportingSpec extends FlatSpec with Matchers with Inspect
 
   val genesis = buildGenesis()
 
-  "ReportingCasper" should "behave the same way as MultiParentCasper" ignore effectTest {
+  "ReportingCasper" should "behave the same way as MultiParentCasper" in effectTest {
```

Comment:
@nzpr you disabled this test. I suppose it's because of replay mismatch (?).

Reply:
@tgrospic yes, as it was failing randomly and messing up merge

Reply:
@tgrospic @nzpr  The test fail because of the additional produce or consume event on persistent produce or persistent consume. The test needs to be changed only compare comm events length otherwise it would fail randomly.